File: scripts/fix_import.py
import re

path = 'app/src/main/java/com/mad/techfix/ui/messages/MessagesActivity.java'
with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('com.mad.techfix.api.RetrofitClient', 'com.mad.techfix.network.RetrofitClient')

with open(path, 'w', encoding='utf-8') as f:
    f.write(content)

print("RetrofitClient import fixed")

File: scripts/fix_import.js
const fs = require("fs");
let code = fs.readFileSync("app/src/main/java/com/mad/techfix/ui/admin/profile/ManagerProfileFragment.java", "utf8");
code = code.replace("import com.mad.techfix.utils.SessionManager;", "import com.mad.techfix.data.SessionManager;");
fs.writeFileSync("app/src/main/java/com/mad/techfix/ui/admin/profile/ManagerProfileFragment.java", code);

